docs/reference/content/getting-started/quick-tour.md

```diff
@@ -0,0 +1,206 @@
+every field's value in to a document bound for the database.
+
+The simplest of the two remaining annotations is `@Property`.  This annotation is entirely optional.  If you leave this annotation off, 
+Morphia will use the Java field name as the document field name.  Often times this is fine.  However, some times you'll want to change 
+the document field name for any number of reasons.  In those cases, you can use `@Property` and pass it the name to be used when this 
+class is serialized out to a document to be handed off to mongodb.  
+
+This just leaves `@Reference`.  This annotation is telling Morphia that this field refers to other Morphia mapped entities.  In this case 
+Morphia will store what mongodb calls a `DBRef` which is just a collection name and key value.  These referenced entities must already be
+ saved or at least have an ID assigned or Morphia will throw an exception.
+ 
+## Saving Data
+
+For the most part, you treat your Java objects just like you normally would.  When you're ready to write an object to the database, it's 
+as simple as this:
+
+```java
+final Employee elmer = new Employee("Elmer Fudd", 50000.0);
+datastore.save(elmer);
+```
+
+Taking it one step further, lets define some relationships and save those, too.
+
+```java
+final Employee daffy = new Employee("Daffy Duck", 40000.0);
+datastore.save(daffy);
+
+final Employee pepe = new Employee("Pepé Lepew", 25000.0);
+datastore.save(pepe);
+
+elmer.getDirectReports().add(daffy);
+elmer.getDirectReports().add(pepe);
+
+datastore.save(elmer);
+```
+
+As you can see, we just need to create and save the other Employees then we can add them to the direct reports list and 
+save.  Morphia takes care of saving the keys in Elmer's document that refer to Daffy and Pepé.  Updating data in mongodb is as simple as 
+updating your Java objects and then calling `datastore.save()` with them again.  For bulk updates (everyone gets a raise!) this is not 
+the most efficient way doing updates.  It is possible to update directly in the database without having to pull in every document, 
+convert to Java objects, update, convert back to a document, and write back to mongodb.  But in order to show you that piece, first we need
+ to 
+see 
+how to query.
+
+## Querying
+
+Morphia attempts to make your queries as type safe as possible.  All of the details of converting your data are handled by Morphia 
+directly and only rarely do you need to take additional action.  As with everything else, `Datastore` is where we start:
+
+```java
+final Query<Employee> query = datastore.createQuery(Employee.class);
+final List<Employee> employees = query.asList();
+```
+
+This is a basic Morphia query.  Here, we're telling the `Datastore` to create a query that's been typed to `Employee`.  In this 
+case, we're fetching every `Employee` in to a `List`.  Ignoring the obvious potential for memory errors, this is not usually that helpful
```

Comment:
Most people won't understand what you mean by "obvious potential for memory errors".  And even with a filter, it may be too much data for a List.